app/models/user.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: DataTypes.STRING
+      }
+    },
+    {
+      timestamps: false
```

Comment:
you should mark this option as true. most probably you need to know when a user was created or modified.